src/cls/rgw/cls_rgw_ops.h

```diff
@@ -527,6 +527,19 @@ struct rgw_cls_obj_remove_op {
     decode(keep_attr_prefixes, bl);
     DECODE_FINISH(bl);
   }
+
+  void dump(ceph::Formatter *f) const {
+    f->open_array_section("keep_attr_prefixes_list");
+    for (auto iter = keep_attr_prefixes.begin(); iter != keep_attr_prefixes.end(); ++iter) {
+      f->dump_string("keep_attr_prefix", *iter);
+    }
+    f->close_section();
```

Comment:
the helpers in src/common/ceph_json.h support `std::list<T>` already:
```c++
encode_json("keep_attr_prefixes", keep_attr_prefixes, f);
```

Reply:
ok, missed that encode_json.. Changing it